src/directives/decorators/stb-webmanual/stb-webmanual-decorator.js

```diff
@@ -58,6 +60,8 @@ angular.module('schemaForm').config(['schemaFormDecoratorsProvider', function(de
     radios: base + 'radios.html',
     'radios-inline': base + 'radios-inline.html',
     radiobuttons: base + 'radio-buttons.html',
+    standarttable: base + 'standart-table.html',
+    addrowbutton: base + 'add-row-button.html',
```

Comment:
rename 'addrowbutton' to 'button' or 'actionButton' and do the same with name of file 'add-row-button.html' (e.g. button.html or action-button.html).